### ISSUE TITLE
Memory viewer is missing some keyboard navigation features

### ISSUE THREAD

**Issue by benrg** [cheat-engine/cheat-engine#1762]
There are a lot of keyboard shortcuts in the Memory Viewer window, but a few omissions limit what can be done without a mouse.

* When you press Ctrl+G to go to a new address in the hex view pane, the window scrolls but the cursor remains at the old address, and if you press any cursor key the window jumps back to the old address. It seems to be necessary to click with the mouse after Ctrl+G to avoid this.
* There seems to be no way to select a range of bytes in the hex view pane without the mouse (Shift+arrow keys doesn't work, at least).
* When a range of bytes is selected, the arrow keys scroll and "rotate" the pane, and there seems to be no way to get the cursor back without clicking. (The "rotate" ability is useful, so I hope that some way to do it can be retained if selection with the keyboard becomes possible.)
* When you move the cursor one row above the top visible row in the hex viewer, it doesn't scroll to bring it into view.
* There is no visual indicator of which pane has focus.

Also, Tab doesn't work to switch panes, though it seems to have no other function in either one. Shift+Tab does work, though.

**Comment by Symbai** [comment]
> * When you move the cursor one row above the top visible row in the hex viewer, it doesn't scroll to bring it into view.

It does if you press 'up arrow' key.

**Comment by benrg** [comment]
Not on my machine, Maybe it depends on the font or system metrics in some way.

**Comment by cheat-engine** [comment]
that's a bug.  It does when you press up on the most left byte but not on any other byte until you get to the 2nd row
